acceptance/localcluster/localcluster.go

```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```

Comment:
I think this behavior should be under a flag and default to false or be controlled by an env variable.